lib/instance_billing_flavor_check/utils.py

```diff
@@ -240,25 +240,33 @@ def make_request(rmt_ip_addr, metadata, identifier):
             message = 'Request error:{}'.format(err)
         except Exception as err:
             message = 'Unexpected error: {}'.format(err)
-    
+
         if message:
-            logger.warning(
-                'Attempt {}: failed: {}'.format(retry_count, message)
-            )
-            if 'Timeout' in message:
+            if 'Timeout' in message or 'Connecting' in message:
+                logger.warning(
+                    'Attempt {}: failed: {}'.format(retry_count, message)
+                )
                 retry_count += 1
+                if retry_count == 4:
+                    return
+
                 time.sleep(2)
-            if 'Timeout' in message or retry_count == 4:
-                return
+                continue
+
+            # error is not time out => return None
+            logger.warning('Access failed {}'.format(message))
+            return
 
         if response.status_code == 200:
             result = response.json()
             logger.debug(result)
             return result.get('flavor')
 
-    logger.error(
-        'Request to check if instance is PAYG/BYOS failed: %s', response.reason
-    )
+        logger.error(
```

Comment:
Remove the error

Reply:
what happens if the request finishes properly and we get a non 200 http status code ?
I think we should keep that message

Reply:
200